activity3/assets/style.css

```diff
@@ -0,0 +1,86 @@
+* {
+    box-sizing: border-box;
+}
+body, h1 {margin: 0;}
+body {
+    font-family: monospace;
+    line-height: 1.4em;
+    color: #333;
+    font-size: 20px;
+    background-color: rgb(147, 109, 79);
+}
+.container {
+    max-width: 1200px;
+    height: auto;
+    margin: auto;
+}
+.p2 {
+    padding: 20px;
+}
+.flex {
```

Comment:
Kung nagimport ka na ng bootstrap dimo need ioverride yung flex class since this is built in. If you wannt to have custom flex recreate or customize it by using other class